RecoTracker/LSTCore/standalone/setup.sh

```diff
@@ -7,18 +7,18 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 source $DIR/code/rooutil/thisrooutil.sh
 
 ARCH=$(uname -m)
-if [[ $(hostname) == *lnx4555* ]]; then
-  export SCRAM_ARCH=el9_amd64_gcc12
-elif [[ $ARCH == "aarch64" || $ARCH == "arm64" ]]; then
-  export SCRAM_ARCH=el9_aarch64_gcc12
+if [ -z ${CMSSW_SEARCH_PATH+x} ]; then
+  if [ -z ${CMSSW_VERSION+x} ]; then
```

Comment:
I changed it to check if a version was set in case we want to override it in the CI. Also, now it uses the command Slava suggested instead of a hard-coded path for the release in case we want to use an IB version.

Reply:
Actually, now I'm thinking that this might lead to confusion. It should check for something more explicit like `FORCED_CMSSW_VERSION`

Reply:
Fixed